### PR TITLE
feat: merged spin & slight workloads with path prefixes 

### PR DESCRIPTION
**Pull request by Mossaka**
This PR unifies the runtime and workload deployments for spin and slight and directs traffic to the individual workloads by adding a path prefix of `/slight` or `/spin`.

### CODE REVIEW

**Review by devigned — CHANGES_REQUESTED** [review]
@Mossaka looks like there's a conflict.

**Comment by Mossaka** [comment]
> @Mossaka looks like there's a conflict.

will work on this shortly!

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -25,25 +25,18 @@ jobs:
       - name: copy release workload assets into _dist
         run: |
           mkdir -p _dist
-          cp ./deployments/workloads/spin/runtime.yaml _dist/spin_runtime.yaml
-          cp ./deployments/workloads/spin/workload.yaml _dist/spin_workload.yaml
-          cp ./deployments/workloads/slight/runtime.yaml _dist/slight_runtime.yaml
-          cp ./deployments/workloads/slight/workload.yaml _dist/slight_workload.yaml
-      - uses: actions/download-artifact@v3
```
Comment (devigned) [inline .github/workflows/release.yaml:32]: We need the release artifacts downloaded still. Can you add this back in?

**Review by devigned — CHANGES_REQUESTED** [review]
I think there's a bit too much red.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (Mossaka) [reply .github/workflows/release.yaml:32]: oops you're right! That was a mistake. I will add them back

**Review by devigned — APPROVED** [review]
lgtm!

**Pull request merged**